src/proposals/20250306/SparkEthereum_20250306.t.sol

```diff
@@ -0,0 +1,168 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.10;
+
+import { Base }     from 'spark-address-registry/Base.sol';
+import { Ethereum } from 'spark-address-registry/Ethereum.sol';
+
+import { IMetaMorpho, IMorpho, MarketAllocation, MarketParams } from 'metamorpho/interfaces/IMetaMorpho.sol';
```

Comment:
`MarketAllocation` isn't used anywhere. Do we need it?

Reply:
I can remove

Reply:
Removed